docs/source/model_design/vivarium_model_components/risk_factors/risk_mediation/index.rst

```diff
@@ -30,4 +30,384 @@ Risk factor mediation
+  Since all of the mediation we have modeled so far has been between continuous
+  risk factors, we haven't yet explored how to make different assumptions (neither has GBD).
+
+Approach 1: consistent with RR application, inconsistent with GBD mediation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this approach, we work backwards from the result reported by GBD (the mediation factor)
+to an underlying data point we can use to create an internally-consistent set of causal effects.
+It is an "undo and redo" approach: we undo the calculations GBD did in additive space to infer
+the data they used as inputs, then do a completely different calculation on those data in multiplicative space.
+To do this, we have to know whether the mediation factor for the pathway in question was calculated
+with the adjusted RR method ("Comparing crude RR versus mediator-adjusted RR"
+in `the GBD risk factors methods appendix [supplementary appendix 1] <https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(20)30752-2/fulltext#supplementaryMaterial>`_) or
+the delta method ("Estimating the mediation factor by pathway of the effect").
+
+.. todo::
+
+  Where should someone look to find this out?
+
+Adjusted RR method
+^^^^^^^^^^^^^^^^^^
+
+In this case, GBD's data were a pair of RRs for the risk-outcome pair,
+one adjusted for the mediator and one not.
+We presume that the unadjusted RR is (approximately) equal to the total RR
+GBD reports for the risk-outcome pair.
+Therefore, we manipulate this equation from the GBD methods appendix:
+
+.. math::
+
+  MF = \frac{RR_\text{r→o,unadjusted for m} - RR_\text{r→o,adjusted for m}}{RR_\text{r→o,unadjusted for m} - 1}
+
+to solve for :math:`RR_\text{r→o,adjusted for m}`:
+
+.. math::
+
+  RR_\text{r→o,adjusted for m} = RR_\text{r→o,total} - MF * (RR_\text{r→o,total} - 1)
+
+:math:`RR_\text{r→o,adjusted for m}` is the underlying data point, but it
+isn't particularly useful to us, as it doesn't necessarily show up
+anywhere in our causal diagram.
+We can use it to calculate a delta, an (assumed linear) increase in the mediator
+per unit increase in the mediated risk:
+
+.. math::
+
+  \Delta_\text{r→m} = \frac{log(RR_\text{r→o,total}) - log(RR_\text{r→o,adjusted for m})}{log(RR_\text{m→o,total})}
+
+Every mediation pathway that begins with the same primary risk
+and mediator will give us the information we need to calculate such a linear
+risk-risk effect.
+For example, BMI→SBP→IHD and BMI→SBP→Stroke both imply delta values for BMI→SBP.
+However, the calculated values from different mediation pathways
+may be inconsistent, since GBD does not enforce consistency.
+We will have to choose one or aggregate them, depending on the situation:
+it could be that we trust the GBD modeling of one of the mediation pathways
+more than the others, or that the values are all similar so it doesn't matter
+much what we do.
```

Comment:
Average them?

Reply:
I didn't want to write something we hadn't actually done, and I believe in CVD in this situation we just chose one arbitrarily. I agree that the average would be a slightly more principled approach.

Reply:
Yeah I think in CVD we did just pick one. This was mainly a result of the specific process to get there. Did the math wrong -> deltas looked almost identical -> picked one. Then Zeb figured out the math was wrong -> recalculated -> they did NOT look identical -> kept one similar to the prior values we had to maintain V&V and continuity in modeling. 

I agree averaging might be a good thing to recommend moving forward.

Reply:
Ok given the history and the seeming consensus here, I'll include averaging as a recommendation though note that we haven't actually done it before.